<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhonebookForm</class>
 <widget class="QWidget" name="PhonebookForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string>Przeglądaj książkę telefoniczną</string>
   </property>
   <widget class="QTableView" name="phonesTableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>600</width>
      <height>341</height>
     </rect>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QFrame" name="browseButtonsFrame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>481</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_24">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QToolButton" name="showAllPhonesToolButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Pokaż wszystkie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="searchPhonesToolButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Wyszukiwanie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearPhonesToolButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Wyczyść</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
